lib/components/atoms/Video/types/index.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = {
+  width?: Number,
```

Comment:
is width & height this really a number or string?

Reply:
yes, it should be string, fixed